src/PhpSpreadsheet/Reader/Xls.php

```diff
@@ -2286,8 +2286,8 @@ private function readXf(): void
                     $rotation = $angle;
                 } elseif ($angle <= 180) {
                     $rotation = 90 - $angle;
-                } elseif ($angle == 255) {
-                    $rotation = -165;
+                } elseif ($angle == Alignment::TEXTROTATION_STACK_EXCEL) {
+                    $rotation = Alignment::TEXTROTATION_STACK_PHPSPREADSHEET;
```

Comment:
Switching to using named constants wherever appropriate is definitely a good idea going forward